hbook/ops.go

```diff
@@ -42,8 +48,13 @@ func DivideH1D(num, den *H1D) (*S2D, error) {
 
 		switch {
 		case b2h == 0 || (b1h == 0 && b1herr != 0): // TODO(sbinet): is it OK?
-			y = math.NaN()
-			ey = math.NaN()
+			if cfg.ignoreNaN {
+				continue
+			} else {
+				y = cfg.replaceNaN
+				ey = 0.0 // TODO(rmadar): I guess this is the most senstive case
```

Comment:
```suggestion
				ey = 0.0 // TODO(rmadar): I guess this is the most sensitive case
```

also, please remove the TODO at line 41, above.
(as this PR is meant to fix it)